Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.Properties.cs

```diff
@@ -111,6 +116,15 @@ public double CompactPaneLength
         /// <summary>
         /// Gets or sets the Brush to apply to the background of the Pane area of the control.
         /// </summary>
+        public Brush AccentColor
```

Comment:
I'd like to use this property to set the brush for the selection indicator in the same way that it is done in the standard Windows 10 apps (News and Money) so this brush will eventually be used for more than just the button background. That's why I prefer a name that is not tied specifically to the control itself. In Windows 10 apps (News, Money) the pane background is independent of the button background. However, the button background is tied to the selection indicator accent. This is why I prefer AccentColor. 1) It recognizes the Hamburger has an AccentColor and a PaneColor. 2) The Accent Color affects multiple controls rendering, 3) It should not be a gradient or some other brush but a solid color.